web/package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "make && docusaurus start",
-    "build": "make && docusaurus build",
+    "start": "make docker && docusaurus start",
+    "build": "make docker && docusaurus build",
```

Comment:
I can, but I would prefer not to. I think it would bring us back to a situation where we have different inconsistent environments. You can always call :

```bash
make && yarn docusaurus -- start
make && yarn docusaurus -- build
```

How come it takes a while??? Once the image is built, it should all be instant, shouldn't it?

Reply:
But the long build only happens once, right? I will add another entrypoint then, but it defeats partially the point of trying to unify the build environment if we also provide first class workarounds 😊

Reply:
Actually, if we let `make` do one thing and `yarn docusaurus -- start` something else, we have better composability. If we can avoid that `yarn` calls make, to make things explicit, I'd much prefer that.

Is that what you suggest, @rdettai?

Reply:
I think you're confusing `yarn docusaurus -- start` and `yarn start`. The former cannot be overriden, the latter we did to run `make` and then the former.

Reply:
Maybe overthinking this a bit, but I see commands in package.json as part of the api, so if we add shortcuts there, ideally they need to be documented and part of the CI. Nothing worse than an every point that is supposed to help but doesn't work 😉. To bad that this docker environment is too slow for your use case, if it cannot be used for documentation writing that partially defeats its purpose 😅

Reply:
Since Rémi did the heavy lifting here, let's try it as is. We can still discuss workarounds or convenience features later. Okay?